src/pages/Environments.js

```diff
@@ -188,6 +188,14 @@ const Environments = () => {
               ])
             }
           },
+          {
+            size: { basis: 120, grow: 0.2 },
+            headerRenderer: () => 'Location',
+            cellRenderer: ({ rowIndex }) => {
+              const { runtimeConfig: { zone, region } } = filteredRuntimes[rowIndex]
+              return zone ? zone : region
```

Comment:
Nit:

```suggestion
              return zone || region
```

looks a bit more readable to me.